app/1.0/start/toolbox/add-elements.md

```diff
@@ -8,79 +8,67 @@ subtitle: "Build an app with App Toolbox"
 Now that you've added a new view to your application, you can start building
 out the details of that view.
 
-In the process, you'll likely want to turn
-to some off-the-shelf components, for example from the
-[Polymer Element Catalog][catalog] or community catalogs like
-[customelements.io][ceio].
+One of the appeals of the web component approach to web development 
+is the ecosystem. Rather than building everything in your app from scratch,
+you can find and use off-the-shelf elements that suit your needs. Two examples
+of places you can find reusable components are the 
+[Polymer Element Catalog][catalog] (elements built by the Polymer team) and 
+[customelements.io][ceio] (elements built by the web components community). 
 
-## Ensure bower is installed
+In this tutorial you install a third-party component and use it in the new
+page that you created in the last step. The element that you're going to 
+install is called `paper-slider`. It's a simple UI subcomponent that lets you 
+drag a slider bar. 
 
-[Bower][bower] is a front-end package manager which is the most common
-tool used for fetching and managing web components.
+## Install a third-party component
 
-Ensure it is installed by running the following command:
+Usually, when working with third-party components, you'd need to browse the
+catalogs and find one that suits your needs, and then get the command for
```

Comment:
Can we shorten this paragraph? Detracts from the focus on the step you're performing.

Suggest: flip this to the last para of this section, start with:

Install the `paper-slider` element:

command,

paragraph about --save

```
**Finding elements:** Usually, when working with third-party components, you'd browse the
catalogs and find one that suits your needs, and follow the installation instructions there.
{.alert .alert-info}
```

WDYT?